### DISCUSSION TITLE
Blind with Stepper Motor hasn't enough way to close

### DISCUSSION THREAD

**Discussion by InZan3** [arendst/Tasmota#12241] [Q&A] [answered]
I use a nema 17 Stepper with 14/1 Gearbox and everything configured like its done in the Wiki. everything works well but the blind stops after 15cm. when I press the Button for up or down the blind moves, stops and i am not able to move it more in the same direction. how can I solve this?

**Comment by ascillato2** [comment]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you. Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)

**Comment by TrDA-hab** [comment, marked as answer]
https://github.com/arendst/Tasmota/discussions/11731

You need to gradually change (increase) these variables ... and achieve the desired result.
ShutterOpenDuration1 20 // Opening time = 20 seconds (default = 10 seconds).
ShutterCloseDuration1 20 // Closing time = 20 seconds (default = 10 seconds).
